core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.ts

```diff
@@ -0,0 +1,73 @@
+import { of } from 'rxjs';
+
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+
+import { catchError, switchMap } from 'rxjs/operators';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+@Component({
+    selector: 'dot-ai-image-prompt',
+    templateUrl: './ai-image-prompt.component.html',
+    styleUrls: ['./ai-image-prompt.component.scss']
+})
+export class AIImagePromptComponent {
+    form = this.fb.group({
+        promptGenerate: ['', Validators.required],
+        promptAutoGenerate: ['', Validators.required]
+    });
+
+    isFormSubmitting = false;
+    showFormOne = true;
+    showFormTwo = false;
```

Comment:
I think you can use one variable to do this conditional with something like `*ngIf="isMainFormOpen; else formTemplateTwo"`